src/app/services/api.ts

```diff
@@ -94,15 +111,27 @@ export class ApiService {
       });
   }
 
+  /**
+   * Log the user out by reseting the token and remove user from local storage.
```

Comment:
```suggestion
   * Log the user out by resetting the token and remove user from local storage.
```

Reply:
Good catch! I'll update this in the follow-up PR